common/persistence/historyStore.go

```diff
@@ -408,40 +401,44 @@ func (m *historyV2ManagerImpl) readRawHistoryBranch(
 func (m *historyV2ManagerImpl) readHistoryBranch(
 	byBatch bool,
 	request *ReadHistoryBranchRequest,
-) ([]*historypb.HistoryEvent, []*historypb.History, []byte, int, int64, error) {
+) ([]*historypb.HistoryEvent, []*historypb.History, []byte, int, int64, int64, error) {
 
 	dataBlobs, token, dataSize, logger, err := m.readRawHistoryBranch(request)
 	if err != nil {
-		return nil, nil, nil, 0, 0, err
+		return nil, nil, nil, 0, 0, 0, err
 	}
 	defaultLastEventID := request.MinEventID - 1
 
 	historyEvents := make([]*historypb.HistoryEvent, 0, request.PageSize)
 	historyEventBatches := make([]*historypb.History, 0, request.PageSize)
 	// first_event_id of the last batch
 	lastFirstEventID := common.EmptyEventID
+	var batchProcessingError error
+	var events []*historypb.HistoryEvent
```

Comment:
[Nit] can you declare the events variable inside the for loop, just to make it clear that it is not used beyond the scope of it

Reply:
Done